packages/design-system/src/components/Stack/Stack.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs';
+import * as StoriesHorizontal from './StackHorizontal.stories';
+import * as StoriesVertical from './StackVertical.stories';
+import { FigmaImage } from "../../docs";
+
+<Meta title="Components/Stack" />
+
+# Stack (aka "Autolayout")
+
+`StackVertical` and `StackHorizontal` are opinionated layout components. They serve has wrappers for most layout needs that would otherwise require manual CSS.
+
+On Figma, this pattern is found in the Autolayout tool.
+
+However, where Figma lets you pick random margin and padding values, `Stacks` limit those options to design tokens.
+
+It is built around a `StrackPrimitive` private component for composition.
+
+
+## Zoning
+
+<FigmaImage src="https://www.figma.com/file/qc8oUSyVJM67gSsbTShftt/%F0%9F%93%90-Zoning?node-id=1044%3A237" />
+
+## Style
+NA
+
+## States
+NA
+
+## Interactions
+NA
+
+## Content
+`StackVertical` and `StackHorizontal` accept any children as long as they are valid React children types.
+
+If you need to override the positional behavior of a child element, use the `StackItem` component:
+
+```tsx
+<StackVertical gap="XS" as="ul">
+    <li>List entry</li>
+    <StackItem as="li" align="center" grow>
+        List entry 2
+    </StackItem>
+    <li>List entry 3</li>
+</StackVertical>
+```
+
+## Usage
```

Comment:
We need doc for the 3 spacing mode (size vs {x,y} vs {top,bottom,left, right})

For padding or margin on 1 side, you have to specify all values, even the ones you want to leave at 0. The code base the shape to `top` so in reality, it is mandatory to set the top value to have the other sides taken into account.
We should have more precision in the doc to tell user to set all the values in this mode.

Reply:
Sure, well spotted.

Reply:
Done!